es/src/test/java/ru/itmo/es/EsApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package ru.itmo.es;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class EsApplicationTests {
+
+    @Test
+    void contextLoads() {
```

Comment:
юнит-тесты хз как писать при том что я замокать эту штуку не могу
```
org.mockito.exceptions.base.MockitoException: 
Cannot mock/spy class ru.quipy.core.EventSourcingService
Mockito cannot mock/spy because :
 - final class
```
а интеграционные тесты в папку с юнит тестами пихать не очень хорошая идея
при защите пинганем ручки из Postman